pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -160,7 +160,7 @@ backendStdenv.mkDerivation rec {
     if [ -z "\''${CUDAHOSTCXX-}" ]; then
       export CUDAHOSTCXX=${backendStdenv.cc}/bin;
     fi
-    export NVCC_PREPEND_FLAGS+=' --compiler-bindir=${backendStdenv.cc}/bin'
+    export NVCC_PREPEND_FLAGS+=' --compiler-bindir=${backendStdenv.cc}/bin -Xfatbin=-compress-all'
```

Comment:
could we add a comment (in nix) clarifying what this does and why it's important?

Reply:
Will do!